docs/sources/_index.md

```diff
@@ -21,7 +21,7 @@ form programmable observability **pipelines** for telemetry collection,
 processing, and delivery.
 
 {{% admonition type="note" %}}
-This page focuses mainly on [Flow mode][], the Terraform-inspired variant of Grafana Agent.
+This page focuses mainly on [Flow mode]({{< relref "./flow/_index.md" >}}), the Terraform-inspired variant of Grafana Agent.
```

Comment:
```suggestion
This page focuses mainly on Flow mode, the Terraform-inspired variant of Grafana Agent.
```

I'm going to just remove the link here. We link to the Flow mode docs in several places throughout this topic, so we aren't leaving the readers hanging.

We need to avoid using relref where possible because these topics are mounted in the Cloud docs and relrefs like thsi will fail in Cloud (because there's no destination for this relref when the topic is viewed in Cloud docs)